internal/vfs/vfstest/vfstest.go

```
@@ -168,6 +168,29 @@ func (m *mapFS) open(p canonicalPath) (fs.File, error) {
 	return m.m.Open(string(p))
 }
 
+func (m *mapFS) remove(path string) error {
+	canonical := m.getCanonicalPath(path)
+	canonicalString := string(canonical)
+	fileInfo, e := m.m.Stat(canonicalString)
+
+	if e != nil {
+		return e
```

This should return a nil error if the file isn't found.